chaos_wrapper.sh

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+function rollback() {
+    git checkout master
+    git checkout HEAD^
+}
+
+/root/.virtualenvs/chaos/bin/python chaos.py || rollback
```

Comment:
Basically rollback tries only the previous commit (this can be a cool attack vector btw, because previous commit can be in the same PR that crashed it)